dask/tests/test_threaded.py

```diff
@@ -105,58 +103,28 @@ def test_f():
     assert L == [1] * 20
 
 
-code = """
-import sys
-import time
-from dask.threaded import get
-
-def signal_started():
-    sys.stdout.write('started\\n')
-    sys.stdout.flush()
-
-def long_task(x):
-    time.sleep(5)
-
-dsk = {('x', i): (long_task, 'started') for i in range(100)}
-dsk['started'] = (signal_started,)
-dsk['x'] = (sum, list(dsk.keys()))
-get(dsk, 'x')
-"""
-
-
 # TODO: this test passes locally on windows, but fails on appveyor
 # because the ctrl-c event also tears down their infrastructure.
 # There's probably a better way to test this, but for now we'll mark
 # it slow (slow tests are skipped on appveyor).
 @pytest.mark.slow
```

Comment:
Then again it could be the merge commit gotten and the commit the status was associated with mismatch. Perhaps this was the [build]( https://ci.appveyor.com/project/daskdev/dask/build/1.0.1132#L2823 ) that should have been associated with the removal of the skip.